client/src/pages/AdminDashboard.js

```diff
@@ -24,14 +24,31 @@ const AdminDashboard = (props) => {
   const [chartTypes, setChartTypes] = useState(null);
 
   // Volunteers SignedIn By Event Type
-  const [totalVolunteersByEventType, setVolunteersSignedInByEventType] = useState({});
-  const [totalVolunteerHoursByEventType, setVolunteeredHoursByEventType] = useState({});
-  const [totalVolunteerAvgHoursByEventType, setAvgHoursByEventType] = useState({});
+  const [
```

Comment:
Here and below lines that less than 100 symbols shouldn't be formatted. It's not clear why this statement should be divided into multiple lines.

Reply:
(see above)